File: src/main/java/de/maggiwuerze/xdccloader/persistency/ChannelRepository.java
package de.maggiwuerze.xdccloader.persistency;

import de.maggiwuerze.xdccloader.model.entity.Channel;
import org.springframework.data.repository.PagingAndSortingRepository;
import org.springframework.stereotype.Repository;

import java.util.List;


@Repository
public interface ChannelRepository extends PagingAndSortingRepository<Channel, Long> {


    List<Channel> findAll();

}

File: src/main/kotlin/de/maggiwuerze/xdccloader/persistency/ChannelRepository.kt
package de.maggiwuerze.xdccloader.persistency

import de.maggiwuerze.xdccloader.model.Channel
import de.maggiwuerze.xdccloader.model.User
import org.springframework.data.repository.PagingAndSortingRepository
import org.springframework.stereotype.Repository
import java.util.*


@Repository
interface ChannelRepository : PagingAndSortingRepository<Channel, Long> {


}